<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>429</width>
    <height>381</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>세진님의 코인 오토트레이너</string>
  </property>
  <widget class="QLabel" name="btc_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>비트코인 현재가:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="BTC_present_price">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>319</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="deposit_info_btn">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>20</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>조회</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="cash_amount">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>20</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>보유현금</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="btc_amount">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>50</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>보유비트코인</string>
   </property>
  </widget>
  <widget class="QLabel" name="btc_label_2">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>300</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>비트코인 평단가:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="BTC_avg_price">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>319</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="trade_btn">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>20</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>거래시작</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
